### ISSUE TITLE
Detect script-here blocks when using execute command

### ISSUE THREAD

**Issue by dbarnett** [syngan/vim-vimlint#119]
I found that the [typical hacks for executing python2 or python3 conditionally](https://vi.stackexchange.com/questions/10913/elegant-way-to-support-both-python-and-python3-in-vim-plugin) using the `execute` command with an endmarker severely confuse vimlint since it doesn't detect the endmarker part and tries to interpret the lines that follow as vimscript. For example:
```VimL
let l:python_command = has('python3') ? 'python3' : 'python'
  execute l:python_command '<< EOF'
import vim
EOF
```
gives me the vimlint error `[EVP_E492] Not an editor command: import vim`

Would it be possible to have some heuristics to catch cases like that and skip over the heredoc contents?

**Comment by dbarnett** [comment]
Ah, looks like this is a dupe of #47. I didn't find it when searching for existing issues since it's all in Japanese.
